src/neo/SmartContract/Native/Role.cs

```diff
@@ -28,6 +28,11 @@ public enum Role : byte
         /// <summary>
         /// NeoFS Alphabet nodes.
         /// </summary>
-        NeoFSAlphabetNode = 16
+        NeoFSAlphabetNode = 16,
+
+        /// <summary>
+        /// Notary nodes
+        /// </summary>
+        Notary = 128
```

Comment:
I believe it is complex as discussed but may be worth because each time more we need services for multi-signatures aggregation.